courses/machine_learning/deepdive2/feature_engineering/solutions/3_Basic_Feature_Engineering_in_Keras.ipynb

```diff
@@ -0,0 +1,2501 @@
+{
```

Comment:
<p>That's trippy, I didn't know you can combine python and bash like that. Thanks for teaching me a new trick!</p><p> </p><p>I don't see housing_pre-proc.csv. Were you able to get the data approved and uploaded to GCP? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/777/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
I have uploaded the file to my branch here - https://github.com/GoogleCloudPlatform/training-data-analyst/tree/gwendolyn-dev/courses/machine_learning/deepdive2/feature_engineering

The housing dataset is taken from an existing Google colab - [https://colab.sandbox.google.com/notebooks/mlcc/first_steps_with_tensor_flow.ipynb?utm_source=mlcc&utm_campaign=colab-external&utm_medium=referral&utm_content=firststeps-colab&hl=en]
`